app/src/ui/dispatcher/dispatcher.ts

```diff
@@ -2694,13 +2694,50 @@ export class Dispatcher {
     log.info(`[cherryPick] - git reset ${beforeSha} --hard`)
   }
 
+  /** Initializes multi commit operation state for cherry pick if it is null */
+  public resetMultiCommitOperationStateCherryPick(
```

Comment:
Calling this method `reset` makes me think it also deals with clearing out any in-progress state but reading this method it will be a no-op if there's already a multi commit operation in progress. Could/should we rename it to just call it `initialize`?

Looking at `initializeMultiCommitOperationIfConflictsFound` it seems it matches this pattern pretty closely (i.e. no-op if operation in progress).

```suggestion
  public initializeMultiCommitOperationStateCherryPick(
```